### ISSUE TITLE
Corpus documentation tabs use invalid element IDs

### ISSUE THREAD

**Issue by lukavdplas** [UUDigitalHumanitieslab/I-analyzer#1606]
On the [corpus documentation page](https://ianalyzer.hum.uu.nl/info/dbnl), the `id` property of the tab panel is set from the title of the tab, i.e. `General information`, `Citation`, `Licence`. These titles are not always valid DOM IDs.

Proposed solution: add a small transformation function to turn the page title into a valid ID. (Removing spaces should be enough.) Use that to set appropriate properties in the template.
